generic_packer_template.json.j2

```diff
@@ -79,23 +85,29 @@
           ]
         }
     {% endif %}
+    {% if 'concourse_version' in event and event['concourse_version']|length > 0 %}
+        ,
+        {
+          "type": "shell",
+          "inline": [
+            "sudo sh -c 'echo \"export CONCOURSE_VERSION={{ event['concourse_version'] }}\" >> /etc/environment'"
+          ]
+        }
+    {% else %}
+        ,
+        {
+          "type": "shell",
+          "inline": [
+            "echo \"concourse version not set\""
+          ]
+        }
+    {% endif %}
     {% if 'provision_scripts' in event and event['provision_scripts']|length > 0 %}
         {% for provisioner_script in event['provision_scripts'] %}
         ,
         {
           "type": "shell",
           "inline": "{{ provisioner_script }}"
-          {% if 'set_proxy' in event and event['set_proxy'] == true %}
```

Comment:
I thought this was sometimes set to true?

Reply:
it's always set to true.